src/remote-config/remote-config-api-client-internal.ts

```diff
@@ -21,10 +21,16 @@ import { PrefixedFirebaseError } from '../utils/error';
 import * as utils from '../utils/index';
 import * as validator from '../utils/validator';
 import { deepCopy } from '../utils/deep-copy';
-import { ListVersionsOptions, ListVersionsResult, RemoteConfigTemplate } from './remote-config-api';
+import {
+  ListVersionsOptions,
+  ListVersionsResult,
+  RemoteConfigTemplate,
+  RemoteConfigServerTemplateData
+} from './remote-config-api';
 
 // Remote Config backend constants
-const FIREBASE_REMOTE_CONFIG_V1_API = 'https://firebaseremoteconfig.googleapis.com/v1';
+// Honors env param to enable URL override independent of binary change.
+const FIREBASE_REMOTE_CONFIG_URL_BASE = process.env.FIREBASE_REMOTE_CONFIG_URL_BASE || 'https://firebaseremoteconfig.googleapis.com';
```

Comment:
@lahirumaramba I've found it helpful here and in the Web SDK to be able to point the SDK at a local server for testing while in development, and issue reproduction after release. I don't see other references to `process.env` in this SDK, though. Do you have a preferred way to reference an env var?

Reply:
Thanks! This makes sense to me. We use `process.env` to handle other env variables https://github.com/firebase/firebase-admin-node/blob/5fe41189279c0a037db3fb48298a1adc657e9701/src/storage/storage.ts#L48

Let's make a note to document this new env var in our public docs

Reply:
Added a row to our burndown to document this env var (go/rc-ai-burndown).